wp-graphql.php

```diff
@@ -74,8 +74,17 @@ function graphql_init_appsero_telemetry() {
 
 	$client = new Appsero\Client( 'cd0d1172-95a0-4460-a36a-2c303807c9ef', 'WP GraphQL', __FILE__ );
 
-	// @phpstan-ignore-next-line
-	$client->insights()->add_plugin_data()->init();
+	// If the Appsero client is version 1.2.0 or higher, add plugin data
+	if (version_compare( $client->version, '1.2.1', '>=' ) ) {
```

Comment:
Would it be more future proof if we did a `class/method_exists()` instead of `version_compare`, JIC they remove/rename the method in later release - and even if it would be would we care?